samples/dashboards/data/mqtt-connector/demo.js

```diff
@@ -0,0 +1,645 @@
+                    ...mqttConfig,
+                    topic: 'prod/DEMO_Highsoft/kraftverk_2/overview',
+                    ...connConfig
+                }
+            }]
+        },
+        components: [{
+            renderTo: 'column-chart-1',
+            type: 'Highcharts',
+            connector: {
+                id: 'mqtt-data-1',
+                columnAssignment: [{
+                    seriesId: 'power',
+                    data: ['time', 'power']
+                }]
+            },
+            sync: {
+                highlight: true
+            },
+            chartOptions: {
+                title: {
+                    text: 'Power Station 1'
+                }
+            }
+        }, {
+            renderTo: 'data-grid-1',
+            type: 'DataGrid',
+            connector: {
+                id: 'mqtt-data-1'
+            },
+            sync: {
+                highlight: true
+            },
+            dataGridOptions: dataGridOptions
+        }, {
+            renderTo: 'column-chart-2',
+            type: 'Highcharts',
+            connector: {
+                id: 'mqtt-data-2',
+                columnAssignment: [{
+                    seriesId: 'power',
+                    data: ['time', 'power']
+                }]
+            },
+            sync: {
+                highlight: true
+            },
+            chartOptions: {
+                title: {
+                    text: 'Power Station 2'
+                }
+            }
+        }, {
+            renderTo: 'data-grid-2',
+            type: 'DataGrid',
+            connector: {
+                id: 'mqtt-data-2'
+            },
+            sync: {
+                highlight: true
+            },
+            dataGridOptions: dataGridOptions
+        }, {
+            renderTo: 'html-log',
+            type: 'HTML',
+            title: 'MQTT Event Log',
+            html: '<pre id="log-content"></pre>'
+        }, {
+            renderTo: 'app-control',
+            type: 'HTML',
+            html: '<button id="clear-log">Clear log</button>' +
+                '<span id="connect-status">disconnected</span>'
+        }],
+        gui: {
+            layouts: [{
+                rows: [{
+                    // For power station 1
+                    cells: [{
+                        id: 'column-chart-1'
+                    }, {
+                        id: 'data-grid-1'
+                    }]
+                }, {
+                    // For power station 2
+                    cells: [{
+                        id: 'column-chart-2'
+                    }, {
+                        id: 'data-grid-2'
+                    }]
+                }, {
+                    // Log/info area
+                    cells: [{
+                        id: 'html-log'
+                    }]
+                }, {
+                    // Application control
+                    cells: [{
+                        id: 'app-control'
+                    }]
```

Comment:
I don't recommend this as part of the dashboard gui. It's just support stuff, and shouldn't be part of the actual dashboard. If people take this code and enable EditMode they can mess it up big time. Suggest you add it as "manual" markup below `<div id="container"></div>`.

Reply:
Moved out of Dashboards container.